test/functional/Java9andUp/src_latest/org/openj9/test/modularity/tests/MethodVisibilityTests.java

```diff
@@ -0,0 +1,154 @@
+/*******************************************************************************
+ * Copyright (c) 2018, 2018 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+package org.openj9.test.modularity.tests;
+
+import java.lang.module.Configuration;
+import java.lang.module.ModuleFinder;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Optional;
+import java.util.ServiceLoader;
+import java.util.Set;
+
+import org.openj9.test.modularity.common.VisibilityTester;
+import org.testng.annotations.Test;
+import org.testng.log4testng.Logger;
+import static org.testng.Assert.fail;
+import static org.testng.Assert.assertTrue;
+
+
+@Test(groups = { "level.extended" })
+
+public class MethodVisibilityTests {
+
+	private static final String ORG_OPENJ9TEST_MODULARITY = "org.openj9test.modularity";
+	/* Note: module name components should avoid terminal digits. */
+	private static final String MODULE_TEST = ORG_OPENJ9TEST_MODULARITY + ".testerModule";
+	public static final String MODULE_A = ORG_OPENJ9TEST_MODULARITY + ".moduleA";
+	public static final String MODULE_B = ORG_OPENJ9TEST_MODULARITY + ".moduleB";
+	public static final String MODULE_C = ORG_OPENJ9TEST_MODULARITY + ".moduleC";
+	private static final String visibilityTesterFullyQualifiedName = 
+			"org.openj9.test.modularity.tests.VisibilityTesterImpl";
+	private static final String MODULARITY_CODE = 
+			System.getProperty("org.openj9.test.module.path", "/team/pdbain/projects/modularity/code/");
```

Comment:
I will ignore makefile as it will be removed, but there is reference for `/team/pdbain`. Will it be removed?

Reply:
Thanks for catching that. I'll delete the makefile too.  Stand by for an updated changeset.